libsignal-service/src/push_service.rs

```diff
@@ -119,9 +119,9 @@ pub struct PreKeyStatus {
 }
 
 #[derive(Clone)]
-pub struct HttpCredentials {
-    pub username: String,
-    pub password: String,
+pub enum HttpAuth {
```

Comment:
May I suggest a manual implementation of `Debug` here, that prints *only the username* (and possibly "password: [..]" or something like that)?